modules/nf-core/delly/call/tests/main.nf.test

```diff
@@ -0,0 +1,140 @@
+nextflow_process {
+
+    name "Test Process DELLY_CALL"
+    script "../main.nf"
+    process "DELLY_CALL"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "delly"
+    tag "delly/call"
+
+    test("homo_sapiens - [bam,bai]") {
+
+        when {
+            process {
+                """
+
+                input = [
+                    [
+                        [ id:'test' ],
+                        file(params.test_data['homo_sapiens']['illumina']['test_paired_end_recalibrated_sorted_bam'], checkIfExists: true),
+                        file(params.test_data['homo_sapiens']['illumina']['test_paired_end_recalibrated_sorted_bam_bai'], checkIfExists: true),
+                        [],
+                        [],
+                        [],
+                        ],
+                        file(params.test_data['homo_sapiens']['genome']['genome_21_fasta'], checkIfExists: true),
+                        file(params.test_data['homo_sapiens']['genome']['genome_21_fasta_fai'], checkIfExists: true)
+                    ]
```

Comment:
Can you make this a little more clear by working with `input[0]`, `input[1]` and `input[2]`. Also while you are working on this module, can you add meta maps to the `fasta` and `fai` input channels?